modules/contentbox/migrations/2023_07_13_092814_v_6_0_0_ContentTemplates.cfc

```diff
@@ -4,49 +4,59 @@
  */
 component {
 
-	function up( schema, qb ){
-		// Content Templates
-		arguments.schema.create( "cb_contentTemplate", ( table ) => {
-			// Columns
-			table.string( "templateID", 36 ).primaryKey();
-			table.dateTime( "createdDate" );
-			table.dateTime( "modifiedDate" );
-			table.boolean( "isDeleted" ).default( false );
-			table.boolean( "isGlobal" ).default( false );
-			table.string( "contentType", 50 );
-			table.string( "name", 255 );
-			table.string( "description", 1000 );
-			table.longText( "definition" );
+	// Include Utils
+	include template="./_MigrationUtils.cfm";
 
-			// Relationships
-			table.string( "FK_authorID", 36 );
-			table.string( "FK_siteID", 36 ).nullable();
-			table
-				.foreignKey( "FK_authorID" )
-				.references( "authorID" )
-				.onTable( "cb_author" );
-			table
-				.foreignKey( "FK_siteID" )
-				.references( "siteID" )
-				.onTable( "cb_site" );
+	function up( schema, qb ){
+		if ( !hasTable( "cb_contentTemplate" ) ) {
```

Comment:
Let's make this conditional an if/else. If the table exists, let's make sure the column typing and lengths ( for the non-key columns ) are correct.  Let's also make sure the `idx_contentType` index gets created.